ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Yes I think a requirement is to have this ownership under Prometheus/CNCF and as we license everything else under Apache 2.0 we should do the same here.